MAUI/Calendar/views.md

```diff
@@ -0,0 +1,190 @@
+
+### Week number
+It displays week number for the current view dates in the month view by setting the `ShowWeekNumber` property. By default, the ShowWeekNumber is `false`. If you need to show the week number in the month view by setting the ShowWeekNumber as true. Week numbers will be displayed based on the ISO standard.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" View="Month"> 
+                      <Calendar:SfCalendar.MonthView>
+                        <Calendar:CalendarMonthView ShowWeekNumber="True"/>
+                      </Calendar:SfCalendar.MonthView>>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView = new CalendarMonthView()
+{
+    ShowWeekNumber = true,
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![Week number in .NET MAUI Calendar.](images/views/maui-week-number.png)
+
+
+#### Week number appearance
+Week number Background and TextStyle can be customized in the month view. Background color can be changed by using the [Background] property and the textStyle can be changed by using the [TextStyle] property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar"  View="Month">
+            <Calendar:SfCalendar.MonthView>
+                <Calendar:CalendarMonthView ShowWeekNumber="True">
+                    <Calendar:CalendarMonthView.WeekNumberStyle>
+                        <Calendar:CalendarWeekNumberStyle Background="DeepSkyBlue">
+                            <Calendar:CalendarWeekNumberStyle.TextStyle>
+                                <Calendar:CalendarTextStyle TextColor="White" FontSize="12" />
+                            </Calendar:CalendarWeekNumberStyle.TextStyle>
+                        </Calendar:CalendarWeekNumberStyle>
+                    </Calendar:CalendarMonthView.WeekNumberStyle>
+                </Calendar:CalendarMonthView>
+            </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView = new CalendarMonthView()
+{
+    ShowWeekNumber = true,
+};
+
+CalendarTextStyle textStyle = new CalendarTextStyle()
+{
+    TextColor = Colors.Black,
+    FontSize = 12,
+};
+
+this.Calendar.MonthView.WeekNumberStyle = new CalendarWeekNumberStyle()
+{
+    Background = Colors.DeepSkyBlue,
+    TextStyle = textStyle,
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![week-number-style-in-maui-calendar](images/views/week-number-style-in-maui-calendar.png)
+
+
+## Year view
+The Year view displays the current year's month. A calendar year is a one-year period that begins on January 1 and ends on December 31. By default, initially displays the current years month and the current month is highlighted by a separate color different from the rest of the month color in `Year` view. Can easily navigate to the desired month dates from the year view.
```

Comment:
The Year view displays the current year's month. A calendar year is a one-year period that begins on January 1 and ends on December 31. By default, displays the current year's month and the current month is highlighted by a separate color that is different from the rest of the month color in the `Year` view. You can easily navigate to the desired month dates from the year view.

Reply:
Content modified for the year view